packages/legend-studio/src/models/protocols/pure/v1/transformation/pureGraph/from/V1_MappingTransformer.ts

```diff
@@ -703,7 +721,11 @@ class PropertyMappingTransformer
   visit_XStorePropertyMapping(
     propertyMapping: XStorePropertyMapping,
   ): V1_PropertyMapping {
-    return transformXStorePropertyMapping(propertyMapping, this.context);
+    return transformXStorePropertyMapping(
+      propertyMapping,
+      this.isTransformingEmbeddedPropertyMapping,
```

Comment:
Do we need this flag? I don't think emebdded property mappings can exist as both embedded and xstore property mappings. We should default this to false if thats the case and if its not the case we need a test showing this.